server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Free & Fair Colorado RLA System
+ * 
+ * @title ColoradoRLA
+ * @created Aug 9, 2017
+ * @copyright 2017 Free & Fair
+ * @license GNU General Public License 3.0
+ * @author Daniel M. Zimmerman <dmz@galois.com>
+ * @description A system to assist in conducting statewide risk-limiting audits.
+ */
+
+package us.freeandfair.corla.endpoint;
+
+import javax.persistence.PersistenceException;
+
+import com.google.gson.JsonSyntaxException;
+
+import spark.Request;
+import spark.Response;
+
+import us.freeandfair.corla.Main;
+import us.freeandfair.corla.model.ContestToAudit;
+import us.freeandfair.corla.model.DepartmentOfStateDashboard;
+import us.freeandfair.corla.persistence.Persistence;
+import us.freeandfair.corla.query.DepartmentOfStateDashboardQueries;
+
+/**
+ * The endpoint for selecting the contests to audit.
```

Comment:
@dmzimmerman, just saw this. You are right: that _is_ what we discussed, and I think they can and should be the same.